core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.spec.ts

```diff
@@ -540,4 +540,14 @@ describe('Contributor dashboard Admin page', () => {
     expect(modalSpy).toHaveBeenCalledWith(UsernameInputModal);
     expect(openRoleEditorSpy).toHaveBeenCalledWith('user1');
   }));
+
+  it('should rearrange language name', () => {
+    expect(component.putEnglishLanguageNameAtFront('Hinglish'))
+      .toBe('Hinglish');
+    expect(component.putEnglishLanguageNameAtFront(
+      'بهاس ملايو (Bahasa Melayu)'
+    )).toBe('Bahasa Melayu (بهاس ملايو)');
+    expect(component.putEnglishLanguageNameAtFront('Azeri (Azerbaijani)'))
+      .toBe('Azerbaijani (Azeri)');
```

Comment:
This test only checks to ensure that `component.putEnglishLanguageNameAtFront()` works as intended. However, the method is only being used once to transform data to be displayed in `component.languageChoices`. I think it would be better to write `expect` statement(s) checking what `component.languageChoices` should be as a result from moving the language's English name to the front.

Reply:
Done.